bad-bits/bin/scheduler.js

```diff
@@ -0,0 +1,16 @@
+import { fetchAndStoreBadBits } from '../lib/badbits.js'
+
+export default {
+  // Scheduled handler that runs every 5 minutes
```

Comment:
```suggestion
```

We want to be able to change the interval without having to update this code comment